liferay-echo/liferay-portal
# LPS-188355 Replace nav element with clay:vertical-nav component

Motivation
=======
The motivation behind this ticket is to use the clay:vertical-nav tag in order to standarize the usage of vertical navigation elements.
[LPS-188353](https://liferay.atlassian.net/browse/LPS-188353)
[LPS-188355](https://liferay.atlassian.net/browse/LPS-188355)

**it depends of this one : https://liferay.atlassian.net/browse/LPS-191478

Proposed Solution
========
The proposed solution consists in creating a list of verticalNavItemList and pass it to the clay:vertical-nav.

Steps to Verify
========
For the fragments one go to Product Menu -> Design -> Fragments -> Check that the vertical list looks and works correctly
For the categories one go to Product Menu -> Categorization -> Categories
